assertions/serverIsReachable.ts

```diff
@@ -1,7 +1,48 @@
 import * as isReachable from "is-reachable";
 
 export class ServerIsReachable {
-    public static run(port: number, path: string): Promise<boolean> {
-        return isReachable("http://localhost:" + port + "/" + path);
+    public static async run(parameters): Promise<void> {
```

Comment:
For the parameters you can define an own interface that describes the properties which can be passed.